virttest/utils_sys.py

```diff
@@ -10,6 +10,7 @@
 from avocado.utils import process
 
 
+# todo: check function in avocado.utils after the next LTS
```

Comment:
I'm not quite sure but I think it should be in upper case. TODO Tree(extension of vscode) cannot search for comment tag todo by default.

Reply:
Alternatively we could also go for sphinx compatible `..todo:: ` but I think our approach shouldn't depend on the choice of IDE-s.

Reply:
@Yingshun @pevogam Thanks for the suggestions. I'm going to follow the tradition of 'todo's of avocado-vt which is to replace it with 'TODO'.